cvat-ui/src/components/common/filters/job-filter-configuration-base.ts

```diff
@@ -0,0 +1,84 @@
+// Copyright (C) CVAT.ai Corporation
+//
+// SPDX-License-Identifier: MIT
+
+import { Fields } from '@react-awesome-query-builder/antd';
+import asyncFetchUsers from 'components/resource-sorting-filtering/request-users';
+
+export const fields: Fields = {
+    state: {
+        label: 'State',
+        type: 'select',
+        operators: ['select_any_in', 'select_equals'], // ['select_equals', 'select_not_equals', 'select_any_in', 'select_not_any_in']
+        valueSources: ['value'],
+        fieldSettings: {
+            listValues: [
+                { value: 'new', title: 'new' },
+                { value: 'in progress', title: 'in progress' },
+                { value: 'rejected', title: 'rejected' },
+                { value: 'completed', title: 'completed' },
+            ],
+        },
+    },
+    stage: {
+        label: 'Stage',
+        type: 'select',
+        operators: ['select_any_in', 'select_equals'],
+        valueSources: ['value'],
+        fieldSettings: {
+            listValues: [
+                { value: 'annotation', title: 'annotation' },
+                { value: 'validation', title: 'validation' },
+                { value: 'acceptance', title: 'acceptance' },
+            ],
+        },
+    },
+    dimension: {
+        label: 'Dimension',
+        type: 'select',
+        operators: ['select_equals'],
+        valueSources: ['value'],
+        fieldSettings: {
+            listValues: [
+                { value: '2d', title: '2D' },
+                { value: '3d', title: '3D' },
+            ],
+        },
+    },
```

Comment:
There is no point in this filter so far:
- 3d isn't supported in quality
- tasks in projects can't have different dimensions

Reply:
Removed